receiver/splunkhecreceiver/receiver.go

```diff
@@ -170,6 +171,9 @@ func (r *splunkReceiver) Start(_ context.Context, host component.Host) error {
 
 	mx := mux.NewRouter()
 	mx.NewRoute().HandlerFunc(r.handleReq)
+	if r.metricsConsumer == nil {
```

Comment:
Why is this guarded by checking if metricsConsumer is nil?  Isn't this receiver shared between the metrics and logs pipeline?

Reply:
no, since raw only goes to logs.

Reply:
Shouldn't we check for `logsConsumer != nil` in case we if traces support can be added?

Reply:
OK sounds good. Done.